transport_nantes/mailing_list/templates/mailing_list/status/list_mp.html

```diff
@@ -0,0 +1,25 @@
+<div>
+    <h2>{{div_title}}</h2>
+    <ul>
+        {% for mailing_list,state in base_list %}
+            <li id="id-ml-{{mailing_list.id}}" >
+                <div class="d-flex align-items-center">
+                    {{ mailing_list.mailing_list_name }}
+                    {% if mailing_list.is_petition %}
+                            {% include "mailing_list/status/hidden_form.html"  with button="Signer" %}
```

Comment:
The "button" variable should be renamed "button_label" I think, because this is all it is, a label for the button. Same at Line 15